msal-python-conceptual/advanced/wam.md

```diff
@@ -7,19 +7,16 @@ manager: CelesteDG
 ms.service: msal
 ms.subservice: msal-python
 ms.topic: conceptual
-ms.date: 02/07/2024
+ms.date: 04/24/2025
```

Comment:
Quoting @SHERMANOUKO 's [question](https://github.com/MicrosoftDocs/microsoft-authentication-library-for-python/pull/120#discussion_r2058845211), "We are looking at our Python MSAL docs and refreshing them. Has there been any new feature updates / deprecation in MSAL Python over the last 1 year?"

We [shipped version 1.27 to 1.32](https://github.com/AzureAD/microsoft-authentication-library-for-python/releases) during this time. No deprecation (that is worth mentioning). Major new features include [supporting CIAM's customied domain in v1.28](https://github.com/AzureAD/microsoft-authentication-library-for-python/releases/tag/1.28.0), managed identity support in 1.29, broker-on-mac support (which requires yet another different redirect_uri being registered) in 1.30.

Reply:
> I noticed the oidc_authority parameter when I was syncing our code samples from the one you maintain to the one we use for docs. I updated the identity platform docs to *replace* this change.

Not sure what you meant by "replace this change". Note that the new `oidc_authority` does NOT replace the old `authority`. It is something new for developer to choose from.

Reply:
I meant that we synced the code and updated our content for custom domai to reflect this as per the sample https://github.com/Azure-Samples/ms-identity-docs-code-python/blob/main/flask-web-app/.env.sample.external-id-custom-domain